wallet-new/src/Cardano/Wallet/API/V1/Swagger/Example.hs

```diff
@@ -39,107 +37,15 @@ instance Example a => Example (Maybe a) where
--- to become huge, up to 1000+ bytes, if the 'UnsafeMultiKeyDistr' constructor
--- is used. We do not use this constructor, which keeps the address between
--- ~80-150 bytes long.
-instance Example (V1 Address) where
-    example = fmap V1 . Core.makeAddress
-        <$> arbitrary
-        <*> arbitraryAttributes
-      where
-        arbitraryAttributes =
-            Core.AddrAttributes
-                <$> arbitrary
-                <*> oneof
-                    [ pure Core.BootstrapEraDistr
-                    , Core.SingleKeyDistr <$> arbitrary
-                    ]
+instance Example CUpdateInfo
 
 instance Example (Mnemonic 12)  where
     example = pure def
 
-instance Example (V1 (Mnemonic 12)) where
-    example = V1 <$> example
-
-instance Example Address
-instance Example Metadata
-instance Example AccountIndex
-instance Example WalletId
-instance Example AssuranceLevel
-instance Example SyncPercentage
-instance Example BlockchainHeight
-instance Example LocalTimeDifference
-instance Example PaymentDistribution
-instance Example AccountUpdate
-instance Example Wallet
-instance Example WalletUpdate
-instance Example WalletOperation
-instance Example PasswordUpdate
-instance Example EstimatedFees
-instance Example Transaction
-instance Example WalletSoftwareUpdate
-instance Example NodeSettings
-instance Example SlotDuration
-instance Example WalletAddress
-instance Example NewAccount
-instance Example TimeInfo
-instance Example AddressValidity
-instance Example NewAddress
-instance Example CUpdateInfo
-instance Example SubscriptionStatus
-instance Example NodeId
-
-instance Example InputSelectionPolicy where
-    example = pure OptimizeForHighThroughput
-
-instance Example (V1 InputSelectionPolicy) where
-    example = pure (V1 OptimizeForHighThroughput)
-
-instance Example Account where
-    example = Account <$> example
-                      <*> example -- NOTE: this will produce non empty list
-                      <*> example
-                      <*> pure "My account"
-                      <*> example
-
-instance Example NewWallet where
-    example = NewWallet <$> example
-                        <*> example -- Note: will produce `Just a`
-                        <*> example
-                        <*> pure "My Wallet"
-                        <*> example
-
-instance Example NodeInfo where
-    example = NodeInfo <$> example
-                       <*> example  -- NOTE: will produce `Just a`
-                       <*> example
-                       <*> example
-                       <*> example
-
-instance Example PaymentSource where
-    example = PaymentSource <$> example
-                            <*> example
-
-instance Example Payment where
-    example = Payment <$> example
-                      <*> example
-                      <*> example -- TODO: will produce `Just groupingPolicy`
-                      <*> example
 
 instance Example WalletStateSnapshot
 
 
-
```

Comment:
What about removing this whole file, moving the remaining instances in places where they belong :thinking:  ?

Reply:
@KtorZ The remaining ones are 
- `WalletSnapshot` from *../wallet/src/Pos/Wallet/Web/Methods/Misc.hs*
- `CUpdateInfo` from *../wallet/src/Pos/Wallet/Web/ClientTypes/Types.hs*
- `Mnemonic` from *../wallet/src/Pos/Util/Mnemonic*

Can we really depend on *wallet-new* from the old *wallet*?

Then, also, the class itself, and instances for `()`, `NonEmpty`, `[]`, `Maybe` and `Map` need to be somewhere.

So it would seem difficult…?

Reply:
Hmmmm... Fair point. Not sure why we have this `Example` in the `wallet-new` in a first place since it's quite an isolated module. But it doesn't make sense to import anything from the wallet in the old one,